Automattic/wp-calypso
# Performance Profiler: add badge to highest impact insights

Closes https://github.com/Automattic/dotcom-forge/issues/9019

## Proposed Changes

* add a badge to the highest impact insights

## Why are these changes being made?


https://github.com/Automattic/dotcom-forge/issues/9019 

## Testing Instructions



* Go to `/speed-test` and start a new test, or go to a results page eg. `/speed-test-tool?url=https%3A%2F%2Fwordpress.com%2F&hash=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpZCI6NzA1N30.7KXaBL1qcm0y0GqvM8ymJ0pbST9ls6P09_hri8EaKSI`
* Check the styling of the insights table on both mobile and desktop.

| Viewport | Before | After |
|--------|--------|--------|
| Mobile | ![CleanShot 2024-09-05 at 17 42 15@2x](https://github.com/user-attachments/assets/c1f55b5e-aa92-4ca5-9144-8f62ad275765) | ![CleanShot 2024-09-05 at 17 40 12@2x](https://github.com/user-attachments/assets/cc024427-c6c1-4108-ab9f-87fe72f81172) |
| Desktop | ![CleanShot 2024-09-05 at 17 40 46@2x](https://github.com/user-attachments/assets/7d120cdb-9225-4279-ae39-639167627364) | ![CleanShot 2024-09-05 at 17 41 13@2x](https://github.com/user-attachments/assets/b747c7f0-457d-4f4c-83f8-6b75e826c527) | 

## Pre-merge Checklist
